addons/test_website/static/tests/tours/website_page_properties.js

```diff
@@ -168,7 +144,6 @@ function testCommonProperties(url, canPublish, modifiedUrl = undefined) {
             stepUtils.goToUrl(getClientActionUrl("/")),
             ...assertPageCanonicalUrlIs("/"),
             stepUtils.goToUrl(getClientActionUrl(url)),
-            stepUtils.waitIframeIsReady(), // Necessary if it's the last step of the tour
```

Comment:
Not entirely sure that one was to be removed @SergeBayet, the "checkTorndown" steps are completed with extra steps afterwards so it might be ok, but with or without it... should not there be always some kind of wait after an URL redirection? Arguable I would say that `waitIframeIsReady` was actually not even enough and we should check we landed on the right page inside the iframe.

@pipu-odoo The `goToUrl` utils seems to click on a link with the given url and that's it: is there still something else that checks that the page is being left and prevents further steps to be evaluated while the page is unloading/loading?

Anyways, runbot is green, I'll not delay the merge for this but I am interested to know :pray: